scripts/seeding_management.py

```
@@ -1,30 +1,125 @@
+def apply_seed_time(client, config):
+    """
+    Apply seeding time limits based on tracker configuration.
+
+    DEPRECATED: Use apply_seed_limits() instead for better performance.
+    This function is kept for backward compatibility.
+    """
+    logging.warning("apply_seed_time() is deprecated. Use apply_seed_limits() to apply both limits in one pass.")
+    apply_seed_limits(client, config)
+
+
+def apply_seed_ratio(client, config):
```

Ruff (0.14.3)

Unused function argument: `client`

(ARG001)